### PR TITLE
Add "Fully Working Application" for Flaskex

### PR DESCRIPTION
**Pull request by styfle**
Demo: https://flaskex-postgresql.now-examples.now.sh

### CODE REVIEW

File: flaskex-postgresql/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Alternatively, your new Flaskex app can be automatically deployed and aliased using [Now for GitHub](https://zeit.co/docs/v2/integrations/now-for-github) or [Now for GitLab](https://zeit.co/docs/v2/integrations/now-for-gitlab). Pushing these files to a new repository with a `now.json` file in the root, and with either [Now for GitHub](https://zeit.co/docs/v2/integrations/now-for-github) or [Now for GitLab](https://zeit.co/docs/v2/integrations/now-for-gitlab) configured for that repository, means your site will be automatically deployed for every push and pull/merge request, and aliased for every push to the default branch!
+
+## Included In This Starter
+
+This starter project includes:
+- A `/index.py` file that responds to the `/` route.
+- A `/signup.py` file that responds to the `/signup` route.
+- A `/settings.py` file that responds to the `/settings` route.
+- A `/logout.py` file that responds to the `/logout` route.
+- A `requirements.txt` file that installs dependencies such as Flask, SQLAlchemy, bcrypt, pg8000 (posgresql driver)
```
Comment (styfle) [inline flaskex-postgresql/README.md]:
```suggestion
- A `requirements.txt` file that installs dependencies such as Flask, SQLAlchemy, bcrypt, pg8000 (PostgreSQL driver)
```

File: flaskex-postgresql/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- User settings
+- Bulma CSS framework
+
+To get started with this project yourself, use the following command from [Now CLI](https://zeit.co/docs/v2/getting-started/installation#now-cli):
+
+```shell
+$ now init flaskex-postgresql
+```
+
+> Alternatively, create a project, and Git repository, with this example template [using the ZEIT dashboard](https://zeit.co/new/flaskex-postresql).
```
Comment (styfle) [inline flaskex-postgresql/README.md]:
```suggestion
> Alternatively, create a project, and Git repository, with this example template [using the ZEIT dashboard](https://zeit.co/new/flaskex-postgresql).
```

File: manifest.json [hunk 3]
```diff
@@ -189,6 +189,12 @@
       "path": "/nodejs-canvas-partyparrot",
       "demo": "https://ppaas.now.sh",
       "description": "A Party Parrot generator built on node-canvas"
+    },
+    {
+      "example": "Flaskex with PostgreSQL",
+      "path": "/flaskex",
+      "demo": "https://flaskex.now-examples.now.sh",
```
Comment (styfle) [inline manifest.json]:
```suggestion
      "demo": "https://flaskex-postgresql.now-examples.now.sh",
```

File: manifest.json [hunk 4]
```diff
@@ -189,6 +189,12 @@
       "path": "/nodejs-canvas-partyparrot",
       "demo": "https://ppaas.now.sh",
       "description": "A Party Parrot generator built on node-canvas"
+    },
+    {
+      "example": "Flaskex with PostgreSQL",
+      "path": "/flaskex",
```
Comment (styfle) [inline manifest.json]:
```suggestion
      "path": "/flaskex-postgresql",
```

File: flaskex-postgresql/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# Flaskex with PostgreSQL on ZEIT Now
+
+[Demo](https://flaskex-postgresql.now-examples.now.sh)
+
+This directory a serverless example of [Flaskex](https://github.com/anfederico/Flaskex), ready to deploy on [ZEIT Now](https://zeit.co/now).
```
Comment (msweeneydev) [inline flaskex-postgresql/README.md]:
```suggestion
This directory is a serverless example of [Flaskex](https://github.com/anfederico/Flaskex), ready to deploy on [ZEIT Now](https://zeit.co/now).
```

File: flaskex-postgresql/README.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+To get started with this project yourself, use the following command from [Now CLI](https://zeit.co/docs/v2/getting-started/installation#now-cli):
+
+```shell
+$ now init flaskex-postgresql
+```
+
+> Alternatively, create a project, and Git repository, with this example template [using the ZEIT dashboard](https://zeit.co/new/flaskex-postgresql).
+
+Once initialized locally, you will see several `.py` files which each correspond to a route in defined in the `now.json` configuration file. Each `.py` file is an input to the [Python Builder](https://zeit.co/docs/v2/deployments/official-builders/python-now-python) which emits a corresponding Lambda for each file.
```
Comment (msweeneydev) [inline flaskex-postgresql/README.md]:
```suggestion
Once initialized locally, you will see several `.py` files which each correspond to a route in defined in the `now.json` configuration file. Each `.py` file is an input to the [Python Builder](https://zeit.co/docs/v2/deployments/official-builders/python-now-python) which emits a corresponding [Lambda](https://zeit.co/docs/v2/deployments/concepts/lambdas/) for each file.
```

File: flaskex-postgresql/README.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+You will need to add a couple [secrets](https://zeit.co/docs/v2/deployments/environment-variables-and-secrets) before deploying.
```
Comment (msweeneydev) [inline flaskex-postgresql/README.md]:
```suggestion
You will need to add a couple of [secrets](https://zeit.co/docs/v2/deployments/environment-variables-and-secrets) before deploying.
```

**Comment by styfle** [comment]
@msweeneydev Thanks, fixed!

**Pull request merged**